lib/cryptoexchange/exchanges/neraex/services/market.rb

```diff
@@ -0,0 +1,40 @@
+module Cryptoexchange::Exchanges
+  module Neraex
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          output = super(ticker_url(market_pair))
+          adapt(output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          base = market_pair.base.downcase
+          target = market_pair.target.downcase
+          "#{Cryptoexchange::Exchanges::Neraex::Market::API_URL}/tickers/#{base}#{target}"
+        end
+
+        def adapt(output, market_pair)
+          ticker           = Cryptoexchange::Models::Ticker.new
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Neraex::Market::NAME
+          ticker.last      = NumericHelper.to_d(HashHelper.dig(output, 'ticker', 'last'))
```

Comment:
ruby's `dig` is implemented in **2.3.0**. it makes reading long hash/array easier. also it if safer to dig into unknown (it does not raise exception). so it became my habit since **2.3.0** 😄 

ref: https://www.tiagoamaro.com.br/2016/08/27/ruby-2-3-dig/

what do you think?

Reply:
Got it, it is kinda like `try`